### ISSUE TITLE
Language does not change

### ISSUE THREAD

**Issue by Tsunayoshi** [Noisyfox/ACT.FoxTTS#15]
Hello, I changed the language to English and restarted ACT but the language does not change:
![image](https://github.com/Noisyfox/ACT.FoxTTS/assets/5128201/b6c877ef-480f-4c27-8f88-0d97bd7906ab)

ACT Translate is also installed and it works there.
![image](https://github.com/Noisyfox/ACT.FoxTTS/assets/5128201/85dcdb28-c28c-4ff2-8443-19ef60d4181f)
![image](https://github.com/Noisyfox/ACT.FoxTTS/assets/5128201/c9789cf6-2b8f-43f4-9687-4d95a92eeb5f)

Also do I need translate to use the tts only?

**Comment by Noisyfox** [comment]
Make sure to put the language folder under the same folder as ACT's main exe.

**Comment by Tsunayoshi** [comment]
Yeah i placed it in the root ACT Folder. Weirdly it works for the Translation Plugin but not for Fox.

![image](https://github.com/Noisyfox/ACT.FoxTTS/assets/5128201/b7a4ce93-9ef5-404e-9ddd-ab4a7a5b7c0a)
